tests/ParserTest.php

```diff
@@ -146,6 +146,13 @@ public function colorParsing()
                     'l' => new Size(220.0, '%', true, $oColor->getLineNo()),
                     'a' => new Size(0000.3, null, true, $oColor->getLineNo()),
                 ], $oColor->getColor());
+                $aColorRule = $oRuleSet->getRules('outline-color');
+                $oColor = $aColorRule[0]->getValue();
+                self::assertEquals([
+                    'r' => new Size(0, null, true, $oColor->getLineNo()),
+                    'g' => new Size(0, null, true, $oColor->getLineNo()),
+                    'b' => new Size(0, null, true, $oColor->getLineNo()),
+                ], $oColor->getColor());
```

Comment:
With the change to throw an exception, this would now instead need to assert that there's no `outline-color` rule (i.e. the invalid declaration has been dropped).